README.md

```diff
@@ -55,8 +55,12 @@ apm install nuclide
 npm install -g flow-bin
 ```
 
+## Contributing
+
 For details on pull requests, commit messages and contributing please see [CONTRIBUTING.md](CONTRIBUTING.md)
 
+Unless you explicitly state otherwise in advance, any non trivial contribution intentionally submitted for inclusion in this project by you to the steward of this repository (Zalando SE, Berlin) shall be under the terms and conditions of the [MIT License](LICENSE), without any additional copyright information, terms or conditions.
+
```

Comment:
Do we want that paragraph in BOTH files (`CONTRIBUTIN.md` and `README.md`)?

Reply:
I followed the recommendations from another Zalando OS repository. Technically we don't need the notice here as well, CONTRIBUTING.md should be enough.